versions.yml

```diff
@@ -103,6 +103,20 @@ modGroups:
       desc: Culls particles to improve performance
       url: https://github.com/Commander07/CullParticlesFabric/releases/download/1.1/cullparticles-1.1_1.19.3-1.19.4.jar
 
+    - name: FerriteCore
+      license: MIT
+      homepage: https://modrinth.com/mod/ferrite-core
+      modrinthId: uXXizFIs
+      desc: Optimizes memory usage
+      url: https://cdn.modrinth.com/data/uXXizFIs/versions/FCnCG6PS/ferritecore-6.0.0-fabric.jar
+
+    - name: Starlight
+      license: LGPL-3
+      homepage: https://modrinth.com/mod/starlight
+      modrinthId: H8CaAYZC
+      desc: Rewrites light engine to reduces chunk load stuttering
+      url: https://cdn.modrinth.com/data/H8CaAYZC/versions/98VOoYPX/starlight-1.1.2%2Bfabric.bdaeb21.jar
```

Comment:
This is only reported as 1.20-1.20.1 on Modrinth. Does it work on 1.20.2?

Reply:
I have not personally tested it but Modrinth does say it's for 1.20.2. Screenshot below.

![image](https://github.com/Wynntils/launchy/assets/76391680/41c27ad3-3059-461f-a9d9-3e1e0cb7b097)

Reply:
Ive missed that